content/zh-cn/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -719,9 +712,10 @@ SSH is required if you want to control all nodes from a single machine.
        scp /etc/kubernetes/pki/front-proxy-ca.crt "${USER}"@$host:
        scp /etc/kubernetes/pki/front-proxy-ca.key "${USER}"@$host:
        scp /etc/kubernetes/pki/etcd/ca.crt "${USER}"@$host:etcd-ca.crt
+       # 如果你正使用外部 etcd，忽略下一行
        scp /etc/kubernetes/pki/etcd/ca.key "${USER}"@$host:etcd-ca.key
    done
-   ```
+    ```
    <!--
```

Comment:
I think the added space is not necessary.

And please add a newline before `<!--`.